build/dockerfiles/rhel.install.sh

```diff
@@ -8,7 +8,25 @@
 # SPDX-License-Identifier: EPL-2.0
 #
 
-microdnf install -y findutils bash wget yum gzip tar jq python3-six python3-pip skopeo && microdnf -y clean all && \
+architecture=""
+case $(uname -m) in
+       x86_64) architecture="amd64" ;;
+       s390x) architecture="s390x";;
+esac
+if [ $architecture == "amd64" ]; then \
```

Comment:
shouldn't the line be the same for all archs ? (no if ?)

Reply:
@benoitf If I use the above line then it throws error for s390x for skopeo and jq before enabling the mentioned repo and if I put the same line after enabling the repo then also it throws following error:
```
error: No available modular metadata for modular package 'python36-3.6.8-2.module+el8.0.0+2975+e0f02136.s390x'; cannot be installed on the system
The command '/bin/sh -c /tmp/rhel.install.sh && rm -f /tmp/rhel.install.sh' returned a non-zero code: 1
```
After investigation found that its because rpm version mismatch between Fedora repo and ubi8 repo.
Hence I kept it separate.

Reply:
I mean, to install skopeo/jq later as well even with amd64

Reply:
Following code works for both amd64 and s390x as well
```
case $(uname -m) in
       x86_64) baseurl="http://download.fedoraproject.org/pub/fedora/linux/updates/30/Everything/x86_64/";;
       s390x) baseurl="https://dl.fedoraproject.org/pub/fedora-secondary/releases/30/Everything/s390x/os/";;
esac

microdnf install -y findutils bash wget yum gzip tar python3-six python3-pip && microdnf -y clean all && \

echo "[fedora]
name=Fedora $releasever - $basearch
baseurl=$baseurl
enabled=1
gpgcheck=0" > /etc/yum.repos.d/fedora.repo

microdnf install -y skopeo jq && microdnf update -y skopeo containers-common jq oniguruma && microdnf -y clean all; \
```
Is this the code change you mean?

Reply:
LGTM -- both arches are as close as possible to being the same code.

You could simplify further (instead of the case/esac) and just do something like this:

```
arch="$(uname -m)" && baseurl="http://download.fedoraproject.org/pub/fedora/linux/updates/30/Everything/${arch}/" && if [[ ${arch} != "x86_64" ]]; then baseurl="https://dl.fedoraproject.org/pub/fedora-secondary/releases/30/Everything/${arch}/os/"; fi
```

Which is one line and therefore a bit more embeddable in a Dockerfile, if needed

Reply:
@benoitf @nickboldt Updated PR with the code changes. Please review.